src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -10,39 +10,78 @@
 
 In this section, you will:
 
-- TODO
+- [resource](https://reactnative.dev/docs/debugging)
 
-## Objective 1: TODO
+## Objective: Devtools setup
 
-TODO
+We will be setting up dev tools to help us debug and inspect the UI:
 
-### Concept TODO
+**React Native Debugger**
 
-TODO
+<img alt="Screenshot of react native debugger." src="../../static/img/react-native/07-debugging-devtools/react-native-debugger
```

Comment:
For `alt` text, we always want to explain what is meaningful in the image.

> Screenshot of the React Native JavaScript Inspector interface. It includes common DevTools tabs such as Console, Sources, Memory, and Profiler.

Also, it should have some CSS to constrain it and not let it go underneath the right TOC.